packages/brokeneck-react/src/components/Groups.js

```diff
@@ -19,90 +21,124 @@ import startCase from 'lodash.startcase'
 import useCreateGroupDialog from '../hooks/useCreateGroupDialog'
 import { LOAD_GROUPS } from '../graphql'
 import useFields from '../hooks/useFields'
+import useSurrogatePagination from '../hooks/useSurrogatePagination'
 
 import Square from './Square'
 
+const useStyles = makeStyles(theme => ({
```

Comment:
the layout of this page is extremely similar to that of users. I think it would be a good time to extract something common so we don't have to duplicate so much code between the two. if you decide not to do it in this PR, can you please create an issue to do that so we don't forget?